app/src/main/java/org/simple/clinic/drugs/search/DrugSearchEvent.kt

```diff
@@ -5,3 +5,5 @@ import androidx.paging.PagingData
 sealed class DrugSearchEvent
 
 data class DrugsSearchResultsLoaded(val searchResults: PagingData<Drug>) : DrugSearchEvent()
+
+data class SearchQueryChanged(val searchQuery: String?) : DrugSearchEvent()
```

Comment:
Why does this need to be nullable? 🤔

Reply:
I was trying a different approach for handling the nullable text changes. But we can filter them out at UI level, will make this change.